secure-boot/create_img.sh

```diff
@@ -10,41 +10,59 @@ error_check() {
 
 IMAGELABEL="BAD_INFLUE"
 FILES_DIR="../hello-dasharo/dist"
-IMAGE_FILE="sb_test_BAD_INFLUE.img"
+IMAGE_FILE="sb_test_data.img"
+# 34 megabytes is minimal image size to avoid mkfs.fat complaints on FAT32 creation
+IMAGE_SIZE_MEGABYTES=34
```

Comment:
I would simply make 100MB. If we leave 34MB we can quickly run out of space when the number of tests grows